sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -781,7 +784,19 @@ public readonly partial struct TokenRequestContext
         public bool IsCaeEnabled { get { throw null; } }
         public string? ParentRequestId { get { throw null; } }
         public string[] Scopes { get { throw null; } }
+        int System.Collections.Generic.IReadOnlyCollection<System.Collections.Generic.KeyValuePair<string, object>>.Count { get { throw null; } }
+        object System.Collections.Generic.IReadOnlyDictionary<string, object>.this[string key] { get { throw null; } }
+        System.Collections.Generic.IEnumerable<string> System.Collections.Generic.IReadOnlyDictionary<string, object>.Keys { get { throw null; } }
+        System.Collections.Generic.IEnumerable<object> System.Collections.Generic.IReadOnlyDictionary<string, object>.Values { get { throw null; } }
         public string? TenantId { get { throw null; } }
+        System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<string, object>> System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object>>.GetEnumerator() { throw null; }
+        bool System.Collections.Generic.IReadOnlyDictionary<string, object>.ContainsKey(string key) { throw null; }
+        bool System.Collections.Generic.IReadOnlyDictionary<string, object>.TryGetValue(string key, out object value) { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public static partial class TokenRequestContextFactory
+    {
+        public static Azure.Core.TokenRequestContext FromDictionary(System.Collections.Generic.IReadOnlyDictionary<string, object> dictionary) { throw null; }
```

Comment:
why is this an extension method and not a static method on TokenRequestContext?

Reply:
Just leftover from some experimentation. Fixed.